import datetime

# an attempt, yes
print(datetime.datetime.now())